Framework/Algorithms/test/DiscusMultipleScatteringCorrectionTest.h

```diff
@@ -339,6 +339,20 @@ class DiscusMultipleScatteringCorrectionTest : public CxxTest::TestSuite {
     }
   }
 
+  void test_workspace_containing_spectra_without_detectors() {
```

Comment:
Since this is moving an exception, is it worth adding a test for the failing behaviour to check that an exception _is_ thrown when `xmin` is greater than `xmax`? Or is the exception caught further up?

Reply:
Yes there should be a test for that. I'll add one